assets/github/workflows/build-application.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
           cat abs-path
 
       - name: Initial Setup
-        shell: xonsh {0}
+        shell: bash
```

Comment:
I think that we need to apply these changes to the gitlab also. Please take a look.

Reply:
Sure, will take a look